shared/IConsole.cs

```diff
@@ -15,5 +15,7 @@ public interface IConsole
         bool IsInputRedirected { get; }
         bool IsOutputRedirected { get; }
         bool IsErrorRedirected { get; }
+        ConsoleColor ForegroundColor { get; set; }
```

Comment:
Ah, very good point! Looks like no "regular" NuGet packages are built from this repo, so presumably no one ever references anything here as a DLL reference.